docs/content/en/docs-dev/user-guide/configuration-reference.md

```diff
@@ -183,7 +184,7 @@ spec:
 | labels | map[string]string | Additional attributes of event. This can make an event definition unique even if the one with the same name exists. | No |
 | replacements | [][EventWatcherReplacement](#eventwatcherreplacement) | List of places where will be replaced when the new event matches. | Yes |
 
-## EventWatcherReplacement
+### EventWatcherReplacement
```

Comment:
What are the differences between `##` and `###`?
If everything nested in `##` are `###`, `KubernetesDeploymentInput` and more should be under `###`?

Reply:
You are right! All `###` will be nested under `##`, but the reason why I don't make `KubernetesDeploymentInput` as `###` is because that section and its parent are too far from each other, so that makes it nested there makes no sense. If a section stands right after its parent, then I make it nested, if it is far from its parent, keep it the same as the level of the surrounding section. That's the logic of the current structure.

You can see the difference below.

Before
![Screen Shot 2023-03-30 at 10 01 07](https://user-images.githubusercontent.com/32532742/228717749-51eb577e-2bc6-4d15-98f3-60e262a1c8dc.png)

After
![Screen Shot 2023-03-30 at 10 01 18](https://user-images.githubusercontent.com/32532742/228717772-a9f4f8c0-c876-4a14-a5ca-0a3f818a55a4.png)